### ISSUE TITLE
After reading this paper, I still do not understand how pre-trained language models is to find the label errors.

### ISSUE THREAD

**Issue by guotong1988** [dcx/lnlfm#1]
Thank you very much. @dcx Thank you.

What is the role of pre-trained language models here?

**Comment by guotong1988** [comment]
Oh, I know: "Checking the high loss data points of a foundation model can be useful as a quick and easy data quality health check, or for more rigorous data cleansing." from the [blog](https://dcx.github.io/lnlfm/).

So, the pre-trained language models is fine-tuned on the labeled dataset?

**Comment by guotong1988** [comment]
Oh, "fine-tuned task loss significantly outperforms more complex error-detection mechanisms proposed in previous work."

from the paper abstract.

**Comment by guotong1988** [comment]
“we hypothesize that a low likelihood label is likely to be an error.”
